packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -408,6 +409,10 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
             commandId: TerminalCommands.TERMINAL_CONTEXT.id,
             order: 'z'
         });
+        menus.registerMenuAction(TerminalMenus.TERMINAL_OPEN_EDITORS_CONTEXT_MENU, {
+            commandId: TerminalCommands.TERMINAL_CONTEXT.id,
+            order: 'z'
+        });
```

Comment:
This command is news to me. :-)

Reply:
Yes, same here. Pretty handy. It opens the terminal to the nearest directory path of the file.